requirements/base.txt

```diff
@@ -21,7 +21,7 @@ PyPdf2
 python-twitter==3.2
 sendgrid
 slackclient
-web3
+web3==4.0.0b7
```

Comment:
I think your travis builds are failing due to the dependencies specified for the 4.x beta.  Can you pin to `3.16.5` and see if that resolves the build failures?

Reply:
no.. unfortunately i cant because i wanted to use some of the new syntax / features of web3py4... 

are there dependancy conflicts with other packages in the requirements.txt?  

IIRC i had to `pip install --pre web3` to get this to work..

Reply:
https://github.com/ethereum/eth-abi/issues/25

Reply:
you are a machine.. thanks for reporting that upstream.. and wow, theyve already handled it!! :)